<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMNativeConfig</class>
 <widget class="QWidget" name="WMNativeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Window Manager Native Desktop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="nativeIconsCheckBox">
     <property name="text">
      <string>Use Desktop Icons</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Wallpaper:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nativeWallpaperEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="nativeWallpaperButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
